OneCommunityGlobal/HighestGoodNetworkApp
# Sai preetham porject archive unarchive

# Description
<img width="488" alt="image" src="https://github.com/user-attachments/assets/773b0774-fa96-4a2f-af18-93c949064067">


## Related PRS (if any):
This frontend PR is related to the #XXX backend PR.
…

## Main changes explained:
- Updated Project.jsx with archive and Unarchive button based on isArchive status
- Updated Projects.jsx to fetch project details based on archive /unarchive status for users
- Updated the actions in Project.js to make api call for unarchived projects.
…

## How to test:
1. check into current branch
2. do `npm install` and `...` to run this PR locally
3. Clear site data/cache
4. log as admin/owner  user who has permission to see projects and archive/unarchive
5. go to dashboard->Otherlinks->Projects
6. Confirm whether you can see Show Archived Button and if yes please click on that and confirm that whether you are able to see all archived projects.
7. Please repeat above step vice-versa like after clicking Show Archived Button now see whether you are able to see Hide Archive now try to click on that and see all unarchived projects.
8. Now try to archive any project and after that whether that project is present in archived in project by clicking show archived project and after that try to unarchive the same project and confirm whether it is reflecting in unarchive projects list.

## Screenshots or videos of changes:
